### PR TITLE
Enable nightly builds

### LINKED ISSUE #95: Configure Aurora nightly builds
## Description
We need to get iOS uploaded nightly so we can get people to test Aurora!

## Definition of done
- [x] The apps are built nightly so that a version from master gets uploaded to the AppStore with the current version of master
- [x] The app should be uploaded with a different bundle id to avoid confusion with the current Xamarin app in the Play Store

### PR DESCRIPTION
**Pull request by Odrakir**
Closes #95 

This PR adds a script to set the build number to the commit number in master. It also removes code signing from frameworks.

This together with a new workflow and schedule builds in Bitrise enabled nightly builds. Every day at 3 AM we will send whatever is on master to the AppStore connect.

### CODE REVIEW

File: Modules/API/API.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -450,9 +453,10 @@
 			baseConfigurationReference = 346828705F0E83C5CDCD5EBE /* Pods-API.release.xcconfig */;
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_STYLE = Manual;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = B227VTMZ94;
```
Comment (heytherewill) [inline Modules/API/API.xcodeproj/project.pbxproj:455]: Does this file need these changes? As in: Erasing `DEVELOPMENT_TEAM ` and changing `CODE_SIGN_IDENTITY `

File: Modules/Architecture/Architecture.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -460,9 +460,9 @@
 			baseConfigurationReference = 0B485253630D2DD7C6740D1B /* Pods-Architecture.debug.xcconfig */;
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_STYLE = Manual;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = B227VTMZ94;
```
Comment (heytherewill) [inline Modules/Architecture/Architecture.xcodeproj/project.pbxproj:465]: Same thing

File: Modules/Assets/Assets.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -398,9 +398,10 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Manual;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = B227VTMZ94;
+				DEVELOPMENT_TEAM = "";
```
Comment (heytherewill) [inline Modules/Assets/Assets.xcodeproj/project.pbxproj:404]: Same applies here

File: Modules/Database/Database.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -390,9 +390,10 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_STYLE = Manual;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = B227VTMZ94;
+				DEVELOPMENT_TEAM = "";
```
Comment (heytherewill) [inline Modules/Database/Database.xcodeproj/project.pbxproj:396]: And here

File: Modules/API/API.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Odrakir) [reply Modules/API/API.xcodeproj/project.pbxproj:455]: I don't think we need code signing for all this frameworks that will be embedded in the main app. App Store connect seems to be happy with it.

**Pull request merged**